### ISSUE TITLE
Pylint list of dependencies is incomplete

### ISSUE THREAD

**Issue by bilderbuchi** [ContinuumIO/anaconda-issues#1383]
It seems the list of pylint dependencies is incomplete. This leads to problems like AtomLinter/linter-pylint#197, where the pylint linter plugin for the Atom editor does not find isort, as it's not installed when installing pylint.
`conda info pylint` gives
```
dependencies:
    astroid >=1.4.1,<1.5.0
    python 3.5*
```
When checking against the [list of dependencies in the current pylint source](https://github.com/PyCQA/pylint/blob/2236f185703da86df83b4ca5a9ac944d8bf6e1f1/pylint/__pkginfo__.py#L24-L30), one can see that in addition, 'six', 'isort >= 4.2.5', 'mccabe', 'editdistance' are missing. I got six and mccabe from other package installations, I guess, but editdistance and isort were not installed in my current/up-to-date Anaconda env.

I don't know why not all the dependencies are tracked (at least mccabe and six are available in conda right now). In my case, a `pip install isort` fixed the problem, but I don't know why conda does not take all dependencies into account (and at least warns if they are not available through conda).

**Comment by ilanschnell** [comment]
Thanks for reporting the issue.  I've added the `six` and `isort >=4.2.5` dependencies.  I would consider `mccabe` an optional dependency at this point.  I've tested the `pylint` command with just those dependencies added, and things appear to be working fine.  Adding `mccabe` as a dependency would mean that we also need to pull it into the Anaconda distribution installers (which I want to avoid at this point).
